scrapers/base.py

```diff
@@ -46,3 +47,28 @@ def run(self, since=None):
         :raises NotImplementedError: if the function is not overridden
         """
         raise NotImplementedError()
+
+    def isContainerBuild(self, attached_build):
+        """
+        Check whether a build is a container build.
+
+        :param KojiBuild attached_build: A build attached to an advisory in question
```

Comment:
It seems like this isn't correct. It seems `attached_build` is actually a Teiid result.